### ISSUE TITLE
Export function for downloading a single form

### ISSUE THREAD

**Issue by philerooski** [Sage-Bionetworks/synapseforms#44]
It's a bit complicated downloading a single form since it requires downloading the file handle directly. Let's export a function `get_form` or `download_form` that can take a formDataId (and a formGroupId? Not sure if this is necessary) and return at the least a file path to the unparsed data blob or, even better, a form object with the `$path` attribute added. Example below

```
> get_form(syn = syn, form_data_id = 235)
submitted_forms[[1]]
$formDataId
[1] "235"

$modifiedOn
[1] "2020-05-04T23:19:11.369Z"

$name
[1] "40ea8071-fcb5-4a97-a12f-4a7b8568c3ef"

$createdOn
[1] "2020-05-04T23:19:11.369Z"

$etag
[1] "aa2e1bc3-8e5d-11ea-b3e5-029730b48859"

$createdBy
[1] "3342492"

$dataFileHandleId
[1] "27776867"

$submissionStatus
$submissionStatus$state
[1] "SUBMITTED_WAITING_FOR_REVIEW"

$submissionStatus$submittedOn
[1] "2020-05-04T23:19:12.646Z"

$groupId
[1] "20"

$path
[1] "/var/folders/46/wrl8_mw907d4hq6kcq0p_r4m0000gp/T//Rtmpfo53a2/filee04231488a33.json"
```

**Comment by philerooski** [comment]
@Aryllen I noticed you already have a `download_form_file` function which you don't export, but it takes a pre-signed URL. This function could be a wrapper around `download_form_file` so that we don't have to choose between 

downloading all the forms and storing in a tabular format 
or
generating a pre-signed URL to download the raw form data

**Comment by Aryllen** [comment]
`get_form_temp` is what I was using to download just a single file. It saves it to a temp file and sends back the filename so you can then read it in. I like the idea of an appended path, though.

To download the files, a presigned url is needed (at least it was when I was working on this). To get that url requires the file handle id and form data id. This information should be available with the downloaded metadata. It would be easier to just give the file data id and have the function look up the file handle id, but I am also concerned about repeated calls to get the metadata. This takes a short bit of time that is unnecessary if you already have the information. Maybe an optional parameter for that?

**Comment by philerooski** [comment]
Yes, an optional parameter would be good. The nice thing about implementing `get_form` this way is that it mimics the behavior of `synGet`. We can ignore caching and just download forms to a temp file each time since they're so small.

**Comment by Aryllen** [comment]
Alrighty, sounds good. It seems like a simple update. Not sure when I will get to this, but will add it to my list of To Dos.
